Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -775,6 +769,13 @@ - (void)loadView
         UIButtonConfiguration *cancelButtonConfiguration = [UIButtonConfiguration filledButtonConfiguration];
         [_cancelButton setConfiguration:cancelButtonConfiguration];
 
+        _pipButton = [self _createButtonWithExtrinsicContentSize:buttonSize];
+        [_pipButton setConfiguration:cancelButtonConfiguration];
+        [_pipButton setImage:[UIImage systemImageNamed:@"pip.enter"] forState:UIControlStateNormal];
+        [_pipButton setImage:[UIImage systemImageNamed:@"pip.exit"] forState:UIControlStateSelected];
+        [_pipButton sizeToFit];
+        [_pipButton addTarget:self action:@selector(_togglePiPAction:) forControlEvents:UIControlEventTouchUpInside];
```

Comment:
This code is duplicated between the two branches, I think you can write this once.